lib/devise_security_extension/models/password_expirable.rb

```diff
@@ -51,6 +51,12 @@ def update_password_changed
           self.password_changed_at = Time.now if (self.new_record? or self.encrypted_password_changed?) and not self.password_changed_at_changed?
         end
 
+        def expired_password_after_numeric?
+          return @_numeric if defined?(@_numeric)
+          @_numeric ||= self.expire_password_after.is_a?(1.class) ||
```

Comment:
`||=` is redundant if you're using a guard

Reply:
Good point